docs/src/catalyst_applications/advanced_simulations.md

```diff
@@ -31,13 +31,14 @@ rn = @reaction_network begin
     v0 + hill(X,v,K,n), ∅ --> X
     deg, X --> ∅
 end
+rn = complete(rn)
```

Comment:
I don't think `complete()` here (and in other places are needed). We should also introduce a section somewhere (I am writing right now) explain what `complete` is.